packages/esm-service-queues-app/src/config-schema.ts

```diff
@@ -175,3 +193,14 @@ export interface ConfigObject {
 export interface OutpatientConfig {
   visitTypeResourceUrl: string;
 }
+
+export interface PriorityConfig {
+  conceptUuid: string;
+  tagType: string;
+  tagClassName: 'priorityTag' | 'tag' | null;
```

Comment:
@chibongho .  The `tagType` is the primary way to do this with these tags it seems.  There are 11 types built into carbon for various color combinations:  https://react.carbondesignsystem.com/?path=/docs/components-tag--overview .  My guess is the `tagClassName` was added so that we could have a tag that exactly matched the mockup color scheme.  My feeling is we should just remove it entirely in favor of just using one of the 11 built-in tag types that are carbon "approved", but I was just trying to maintain feature parity here for now.